doc/manual/src/contributing/experimental-flags.md

```diff
@@ -0,0 +1,89 @@
+This section aims at describing the notion of a Nix "experimental
+flag", and how it fits into the big picture of the developpement of Nix.
```

Comment:
```suggestion
This section describes the notion of a Nix's "experimental
features", and how it fits into the big picture of the development of Nix.
```

And rename "experimental/xp flags" to "experimental features" throughout, because the config setting is `experimental-features` after all.

Reply:
Yes, I renamed it to “experimental flag” because it doesn’t have to be a “feature” strictly speaking (because of the section on refactorings). But we can keep the “feature” word for consistency

(Or we could add a new `xp-feature-is-flag` experimental feature to rename the option to `experimental-flags`. But that’s going a bit too meta :wink: )